pkgs/applications/science/logic/coq/default.nix

```diff
@@ -19,8 +20,8 @@ let
    "8.6"	= "1pw1xvy1657l1k69wrb911iqqflzhhp8wwsjvihbgc72r3skqg3f";
   }."${version}";
   coq-version = builtins.substring 0 3 version;
-  buildIde = lablgtk != null;
-  ideFlags = if buildIde then "-lablgtkdir ${lablgtk}/lib/ocaml/*/site-lib/lablgtk2 -coqide opt" else "";
+  camlp5 = ocamlPackages.camlp5_transitional;
```

Comment:
ic, this is specific to `camlp5_transitional`. The others should have been casted with `(ocamlPackages)`.